ui/lib/api.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:gitpods/src/api/api.dart';
+
+class API {
+  API() {
+    // TODO: What's the best way to inject the correct basePath from the backend here?
```

Comment:
Maybe set it by some html attribute on the serverside and read it before initializing the client?

Reply:
Yep. That's my intention, sadly I currently can't figure out how to properly do that on the Dart side. I'll ask on Gitter.